tests/Behat/MailsContext.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Behat;
+
+use Alex\MailCatcher\Behat\MailCatcherContext;
+use Alex\MailCatcher\Message;
+use Behat\Behat\Context\Environment\InitializedContextEnvironment;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Behat\MinkExtension\Context\MinkContext;
+use Behat\MinkExtension\Context\RawMinkContext;
+use Symfony\Component\DomCrawler\Crawler;
+
+class MailsContext extends MailCatcherContext
+{
+    private RawMinkContext $minkContext;
```

Comment:
```suggestion
    private ?RawMinkContext $minkContext = null;
```
Pareil sur les autres contexts dont la variable n'est pas initialisée au constructeur